policies/platformpolicy.md

```diff
@@ -0,0 +1,323 @@
+# Platform Policy
+
+Platforms are classified as "primary", "secondary", "community"
+and "unadopted". Support for a new platform should only be added if it
+is being adopted as a primary, secondary or community platform.
+
+Primary
```

Comment:
I'd expect glossary entries for these.

Why aren't they being moved to the general-policy repo?

Reply:
> I'd expect glossary entries for these.

You mean links to the glossary?  Sure, that could be done, but that's a bit beyond the purpose of this PR.

> Why aren't they being moved to the general-policy repo?

'cause no one has proposed this policy for openssl/general-policies yet...  we did discuss it at some point, but also realised that the table of platforms should not be moved there, as they aren't really a policy.  And no one has done that work yet.  That's also beyond the purpose of this PR.

Reply:
So I went looking in the glossary that we have, and reformatted these definitions to fit